src/excmds.ts

```diff
@@ -4907,4 +4903,13 @@ export async function extoptions(...optionNameArgs: string[]) {
     return winopen("-popup", selectedExtension.optionsUrl)
 }
 
+/** Restores elements killed using ;K. 
+ *
+ *NB: The most recently killed element that has not already been restored will be restored.
+ */
+//#content
+export async function restorekill() {
```

Comment:
Yeah, I don't know what to call this either. In general we have a convention of naming things `[object][action]`, so under that this would be `killrestore` ... but I don't like that either : )

I'll try to think about it a bit more.

Reply:
How about ```resurrect()```, ```revive()``` or ```elementrestore()```?

Reply:
I like elementrestore the most out of those ... still not 100% happy with it though.

I'll try really hard to get round to look seriously at this PR next week : )